tests/unit/skills/people/TestAssets/GetProfileTests/GetProfile_UserNotFound.test.dialog

```diff
@@ -6,6 +6,17 @@
   ],
   "dialog": "People",
   "script": [
+    {
+      "$kind": "Microsoft.Test.UserActivity",
+      "activity": {
+        "type": "event",
+        "name": "SetTestOptions",
+        "value": {
+          "randomSeed": 0,
+          "randomValue": 0
```

Comment:
What's the advantage of doing this over the other?

Reply:
This defaults to the first LG response every time, so for long term it is easier to maintain without having to update the full array of choices every time.